index.html

```diff
@@ -20,7 +20,7 @@ <h2>Download for Minecraft Java Edition 1.8 - 1.12.2</h2>
 							<a href="https://download.cuberite.org/linux-i386/Cuberite.tar.gz">32-bit</a>
 							<a href="https://download.cuberite.org/linux-x86_64/Cuberite.tar.gz">64-bit</a>
```

Comment:
```suggestion
							<a href="https://download.cuberite.org/linux-i386/Cuberite.tar.gz">x86 32-bit</a>
							<a href="https://download.cuberite.org/linux-x86_64/Cuberite.tar.gz">x86 64-bit</a>
```

Now we have an ARM 64 bit, we do actually need to add this, for disambiguation.